projects/m2/021-unique-characters/js/index.js

```diff
@@ -0,0 +1,28 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function calculateUniqueCharacters(text){
+    let uniqueCharactersCount = 0;
+    const uniqueCharactersDictionary = [];
+
+    for (let i = 0; i < text.length; i++) {
+        const character = text.charAt(i);
+
+        if( ! uniqueCharactersDictionary.includes(character) ){        
+            uniqueCharactersDictionary.push(character);
+        } 
+    }
+
+    uniqueCharactersCount = uniqueCharactersDictionary.length;

```

Comment:
The variable I think can be declared and initialized directly here: `const unique.... = ....length`